src/types/inc/viewport.hpp

```diff
@@ -121,6 +121,16 @@ namespace Microsoft::Console::Types
 
         [[nodiscard]] static SomeViewports Subtract(const Viewport& original, const Viewport& removeMe) noexcept;
 
+        constexpr bool operator==(const Viewport& other) const noexcept
+        {
+            return _sr == other._sr;
+        }
+
+        constexpr bool operator!=(const Viewport& other) const noexcept
+        {
+            return _sr != other._sr;
+        }

```

Comment:
Don't ask me why, but without this change the code doesn't compile. 😐

Reply:
_how strange._

Reply:
I've always wondered why we did out-of-body inline operators and didn't just _put them in the classes_